README.md

```diff
@@ -0,0 +1,53 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Purpose of the project here is to master all of the tools and best practices I have learned about in previous steps.
+
+## Built With
+
+- HTML, CSS
+- Webhint
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Git
+- Code Editor. e.g., VS Code
+
+### Setup
+
+- Clone the repository using the `git clone` command.
+
+### Usage
+
+- Open the index.html file.
+
+
+
+## Authors
+
+👤 **Debabrata Mandal (Nayan Karan)**
+
+- GitHub: [@githubhandle](https://github.com/NayanKaran)
+
```

Comment:
- Optional [] kindly add your LinkedIn account here or twitter

Reply:
@vic778 can I merge the project now?

Reply:
@NayanKaran yes you can marge it.

Reply:
Thank you.